firewall/firewall_windows.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package firewall
+
+import (
+	"errors"
+	"fmt"
+
+	log "github.com/cihub/seelog"
+	"github.com/mysteriumnetwork/node/utils"
+)
+
+const (
+	firewallLogPrefix = "[firewall] "
+)
+
+// AddInboundRule adds new inbound rule to the platform specific firewall.
+func AddInboundRule(proto string, port int) error {
+	name := fmt.Sprintf("myst-%d:%s", port, proto)
+	cmd := fmt.Sprintf(`netsh advfirewall firewall add rule name="%s" dir=in action=allow protocol=%s localport=%d`, name, proto, port)
+
+	if inboundRuleExists(name) {
+		return nil
+	}
+
+	_, err := utils.PowerShell(cmd)
```

Comment:
You always swallow output from PowerShell. Not sure of its a good strategy. Maybe this output could explain us some miss execution. Maybe exec command will not return an error, but shell output would basically mean some issue. Maybe we can add this output to Trace level ?

Reply:
If command exited with 0 exit code we print nothing. If the exit code is not 0 we will return it as an error and it will be printed as `Trace`. 
I don't think that we need to print the output of successful commands.

Reply:
We have lots of logs already, it's quite hard to understand them without deep knowledge of `core`. I think we should have proper filtering of log levels (i.e. skip `Trace` by default) before adding even more noise.